pkg/api/types.go

```diff
@@ -2121,6 +2121,16 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
```

Comment:
I don't understand.  You want system pods to use the built-in, special priority.  I'm just saying that the name of that special prio should be something that can't be created by a user (hence the options of names that are invalid resource names)